backend/middleware/authAdmin.js

```diff
@@ -1,5 +1,8 @@
 const CustomError = require("../utils/customError");
 
+// this middleware will check the user is admin or not ,if not return error message.
+// Add this middleware in routes which you want only admin can access
+
 function authAdmin(req, res, next) {
   const role = req.user.role;
   if (role !== "ADMIN") {
```

Comment:
rewrite this to accept a array which we can check in the middleware

Reply:
done take a look